content/reference/admin/private_locations/private_packages/index.md

```diff
@@ -9,71 +9,134 @@ weight: 22033
 
 # Introduction
 
-Using Private Locations allows you to run your simulations in your private cloud. Now, what about your simulations' packages? They may hold private information, so you may want to also store them in your private cloud.
+Private Locations facilitate running simulations in your dedicated cloud environment. 
+Ensure secure storage of sensitive simulation packages in your private cloud. 
 
-Your Control Plane has a private repository feature, allowing you to manage your simulation packages privately, you just have to enable it.
+The Control Plane offers a private repository; enable it for confidential package management!
 
 # Private packages
 
-## Infrastructure
+To create private packages, utilize our Gatling Enterprise Cloud packages API and upload them through a 
+plugin configured with the Control Plane's private package repository and a Configurer public API Token.
```

Comment:
```suggestion
To create private packages, use the Gatling Enterprise Cloud packages API and upload them through a 
plugin configured with the Control Plane's private package repository and a configured public API Token.
```
This sentence seems complicate and the process is described in usage section. Maybe we can write something like 

> Thanks to the control-plane package API and token, you will able to upload directly your package to your control plane. By this way, your package will not leave your infrastructure.